cmake/dependencies.cmake

```diff
@@ -85,7 +100,6 @@ if(TUVX_ENABLE_TESTS)
 
   FetchContent_MakeAvailable(googletest)
 
-  # don't run clang-tidy on google test
-  set_target_properties(gtest PROPERTIES CXX_CLANG_TIDY "")
-  set_target_properties(gtest_main PROPERTIES CXX_CLANG_TIDY "")
```

Comment:
we probably still don't want to run clang-tidy on tests, but maybe this is handled elsewhere?

Reply:
that's my bad; I accidentally deleted that